src/pspm_overwrite.m

```diff
@@ -80,12 +80,14 @@
     % the following code can be re-enabled if the condition can be set
     % -EOF-
     % if feature('ShoverwriteFigureWindoverwrites') % if in gui
-    %   msg = ['Model file already exists. Overwrite?', ...
-    %     newline, 'Existing file: ', fn];
-    %   overwrite = questdlg(msg, ...
-    %     'File already exists', 'Yes', 'No', 'Yes');
-    %   % default as Yes (to overwrite)
-    %   overwrite_final = strcmp(overwrite, 'Yes');
+    if overwrite_final == 2
```

Comment:
If this part of code is wishing to ask users to determine whether to overwrite, the solution shall be `pspm_overwrite`.